geomstats/geometry/spd_matrices.py

```diff
@@ -23,11 +22,10 @@ class SPDMatrices(SymmetricMatrices, EmbeddedManifold):
     def __init__(self, n):
         super(SPDMatrices, self).__init__(
             n=n,
-            dim=int(n * (n + 1) / 2),
-            embedding_manifold=GeneralLinear(n=n))
+            dim=int(n * (n + 1) / 2))
 
     def belongs(self, mat, atol=gs.atol):
-        """Check if a matrix is symmetric and invertible.
+        """Check if a matrix is symmetric with positive eigenvalues.
```

Comment:
Should we say strictly positive here?

Reply:
I think in English positive means French strictly positive, and non-negative means French positive :)